lib/evaluate/fold.cc

```diff
@@ -215,13 +239,13 @@ static inline Expr<TR> FoldElementalIntrinsicHelper(FoldingContext &context,
             // same. Shouldn't this be checked elsewhere so that this is also
             // checked for non constexpr call to elemental intrinsics function?
             context.messages().Say(
-                "arguments in elemental intrinsic function are not conformable"_err_en_US);
+                "Arguments in elemental intrinsic function are not conformable"_err_en_US);
```

Comment:
I see that most compilers tend to start messages with caps, but our messages sometimes do and sometimes don't.  Do we need a suggestion somewhere to favor caps?  The new message at line 579 below may be an exception.

Reply:
Tim's messages tend to start with caps and I've been converting others as I run across them.

Reply:
I've also seen other kinds of inconsistencies.  It would be good to agree on message guidance.

Reply:
I've changed some messages to start with caps.  Watch out for tests ..